cypress/support/execute-tests.sh

```diff
@@ -153,11 +161,18 @@ function overrideSettings() {
   cp -r $SETTINGS_SOURCE $SETTINGS_TARGET
 
   echo 'INFO: writing projection to settings file'
-  sed -i.bak 's@CYPRESS_PROJECTION@'$CYPRESS_PROJECTION'@g' $OPENSPHERE_CONFIG_TESTER && rm $OPENSPHERE_CONFIG_TESTER.bak
```

Comment:
The `-i.bak` flag was for cross-platform support, so we should bring that back for now. Details here:
https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux